### PR TITLE
Add required_terraform pinning Terraform to 0.13.5

### PR DESCRIPTION
**Pull request by danieldreier**
# Goal

Terraform operations should error out if Terraform is run with newer Terraform versions.

## Todos
- [ ] needs to be run / tested

## Reference
@bassrock, @kkyeboah and I discussed this in slack

I tested this in a toy environment but haven't run the `recommendation-api` CDKTF codebase locally

### CODE REVIEW

**Review by mmiermans — APPROVED** [review]
lgtm :+1: Thanks for help fix this!

I'll look to @kkyeboah for a second review.

File: .aws/src/main.ts [hunk 1]
```diff
@@ -265,4 +265,5 @@ class RecommendationAPI extends TerraformStack {
 
 const app = new App();
 new RecommendationAPI(app, 'recommendation-api');
+stack.addOverride("terraform.required_version", "0.13.5")
```
Comment (kkyeboah) [inline .aws/src/main.ts:268]:
```suggestion
const stack = new RecommendationAPI(app, 'recommendation-api');
stack.addOverride("terraform.required_version", "0.13.5")
```

**Review by kkyeboah — CHANGES_REQUESTED** [review]
Thank you for your contribution 💯. We definitely need to require the version.

I do have a small suggestion to fix your addition.

**Comment by mmiermans** [comment]
We've since migrated to Terraform 1.0.11. Not sure if pinning is still necessary.

**Pull request closed without merging**